input/ig.xml

```diff
@@ -632,31 +632,31 @@
 				<reference value="PractitionerRole/bobrester-bob-gp"/>
 			</reference>
 			<name value="PractitionerRole - General practitioner"/>
-			<description value="Dr. Bob Bobrester, GP, at Bobrester Medical Centre"/>
+			<description value="Shows an example for the *AU Core PractitionerRole* profile with practitioner and organisation as references."/>
```

Comment:
here practitioner and organisation are lower case, in the following they are title case. I suspect this should be Practitioner and Organization.